cmd/libsnap-confine-private/infofile.c

```diff
@@ -76,6 +82,31 @@ int sc_infofile_get_key(FILE *stream, const char *key, char **value, sc_error **
         }
         /* Replace the trailing newline character with the NUL byte. */
         line_buf[nread - 1] = '\0';
+
```

Comment:
Can I suggest to use this opportunity to add a bit of code immediately before the section check, to allow comments using `;` or `#` (though ideally that would be any character in a `const char *` "list".

Reply:
While this makes sense and is trivial, I'd prefer to stick to the bug fix here and leave this suggestion for a followup and not grow this PR more (2-3 lines code change, but then 10-20 more lines in the tests).

Reply:
Yeah, sounds a like a followup material to me too.